src/plan/controller_collector_context.rs

```diff
@@ -17,7 +11,6 @@ struct RequestSync {
 pub struct ControllerCollectorContext<VM: VMBinding> {
```

Comment:
It feels like we can change this type to `GCRequest`, as all of its fields and methods are about GC requests.

Reply:
I'll call it `GCRequester` so as not to be confused by a concrete request for collection.  It is an object for making requests.